test/libsolidity/SemanticTest.cpp

```diff
@@ -155,7 +157,14 @@ TestCase::TestResult SemanticTest::run(ostream& _stream, string const& _linePref
 			}
 		}
 
-		if (!success)
+		if (success && !m_runWithYul && compileViaYul)
```

Comment:
Is it possible to make this just a regular expectation failure and thus pressing `u(update)` auto-updates the setting?

Reply:
I will check how to do this, it would be great to have auto update on this feature

Reply:
Figured it out, done with it